### PR TITLE
feat: updated settings - move map

### PR DESCRIPTION
**Pull request by benfurber**
## PR Checklist

- [x] - Commit [messages are descriptive](https://github.com/ONEARMY/community-platform/blob/master/CONTRIBUTING.md#--commit-style-guide), it will be used in our [Release Notes](https://github.com/ONEARMY/community-platform/releases/)
- [x] - Unit and e2e tests for the changes that have been added (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix (fixes an issue)
- [x] Feature (adds functionality)

## What is the current behavior?

Map pins are nested in the main user profile settings tab.

## What is the new behavior?

1. Moves the map pin setting to its own tab
2. Seperate user and map pin stores
3. Increase e2e coverage
4. Fixed impact tab anchoring

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Screenshot
Screenshot 2024-07-24 at 13 31 07

### CODE REVIEW

**Comment by benfurber** [comment]
Still very much a WIP.

**Comment by davehakkens** [comment]
Nice. But would be nice to align a bit more to the design from @dalibormrska which is a bit more organised. Not sure if you need specific explanation or you know what I refer to @benfurber?

Also "location data saved" might be nicer to replace by "Your current map pin is here:"

File: src/pages/UserSettings/content/formSections/SettingsMapPin.section.tsx [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+const LocationDataTextDisplay = ({ location }: ILocationProps) => {
+  if (!location)
+    return (
+      <Text variant="paragraph" data-cy="LocationDataTextDisplay">
+        {mapForm.noLocationLabel}
+      </Text>
+    )
+
+  return (
+    <>
```
Comment (mariojsnunes) [inline src/pages/UserSettings/content/formSections/SettingsMapPin.section.tsx:48]: no need the fragment

**Review by mariojsnunes — APPROVED** [review]

**Comment by benfurber** [comment]
@davehakkens Yes, sorry, went to write a note with my proposed changes. 

When editing a pin, due to how the map loads I realised it's super hard to understand what is currently saved. So I thought displaying it was the easiest option, made good end-to-end testing simply and probably just good practice. Very happy for feedback/improving that, so will update the copy as you suggested and happy for more.

Two fairly held strong opinions on a couple of UX points that I didn't go through with @dalibormrska yet, but happy to be disagreed with:
1. Using a delete icon over the map isn't obvious enough and doesn't make clear it would also clear the description field. I like the icon itself though, could change the button text to include it?
2. The explaination of how to use the map - if you want to help someone use a thing, you have to explain BEFORE the field, not after.

**Comment by davehakkens** [comment]
@benfurber ok that makes sense to me. But there are still a fe additional UI changes.
> 1. Using a delete icon over the map isn't obvious enough and doesn't make clear it would also clear the description field. I like the icon itself though, could change the button text to include it?

Yes adding icon would be nice

2. Page headings and paragraphs are styled different on each page. Different font-size for titles and different font used for the paragraph text. Would be good to align this more, as in the figma file.
image
image
image


3. Would be nice to merge these two text blocks together and remove the blue backround. As in the figma file.
image

**Comment by benfurber** [comment]
Map specific points 1 and 3 in #3801. The consistent point I'll do in a follow-up.

**Pull request merged**